src/tests/multihost/alltests/test_krb_access_provider.py

```diff
@@ -0,0 +1,258 @@
+"""Automation for Krb access provider tests ported from bash
+
+"""
+from __future__ import print_function
+import time
+import pytest
+from sssd.testlib.common.expect import pexpect_ssh
+from sssd.testlib.common.utils import sssdTools
+from constants import ds_instance_name
+
+
+@pytest.fixture(scope='class')
+def custom_setup(session_multihost, setup_sssd_krb, create_posix_usersgroups, krb_connection_timeout):
+    """ Added neccessary sssd domain parameters """
+    tools = sssdTools(session_multihost.client[0])
+    sssd_params = {'sbus_timeout': 30,
+                   'services': "nss, pam",
+                   'config_file_version': 2}
+    tools.sssd_conf('sssd', sssd_params)
+    domain_section = f'domain/{ds_instance_name}'
+    domain_params = {'access_provider': 'krb5',
+                     'use_fully_qualified_names': 'False',
+                     'override_homedir': '/home/%u'}
+    tools.sssd_conf(domain_section, domain_params)
+    tools.clear_sssd_cache()
+
+    sed_cmd = f'sed -i "s/kerberos.example.com/{session_multihost.master[0].sys_hostname}/" /etc/krb5.conf'
+    session_multihost.client[0].run_command(sed_cmd)
+    sed_cmd = 'sed -i "s:^#::g" /etc/krb5.conf'
```

Comment:
What is this line for? It looks like it only removed 'commented' lines in krb5.conf ?